opentracing-spring-web-autoconfigure/src/main/java/io/opentracing/contrib/spring/web/autoconfig/ServerTracingAutoConfiguration.java

```diff
@@ -34,18 +37,32 @@
 public class ServerTracingAutoConfiguration {
     private static final Log log = LogFactory.getLog(ServerTracingAutoConfiguration.class);
 
+    private final ObjectProvider<List<ServletFilterSpanDecorator>> servletFilterSpanDecorator;
```

Comment:
`ObjectProvider` is since 4.3  https://docs.spring.io/spring-framework/docs/current/javadoc-api/org/springframework/beans/factory/ObjectProvider.html. Is there a way to do it without it? We want to support the largest range of spring versions if possible.

Reply:
Spring 4.3 [is part of Spring Boot 1.4.x](https://github.com/spring-projects/spring-boot/wiki/Spring-Boot-1.4-Release-Notes#spring-framework-43) which is the version specified in the [pom.xml](https://github.com/opentracing-contrib/java-spring-web/blob/master/pom.xml#L48)

Reply:
thanks, I was more talking about this https://github.com/opentracing-contrib/java-spring-web/tree/master/opentracing-spring-web package. I forgot that this is auto-configure.